azure-devops/azext_devops/dev/team/invoke.py

```diff
@@ -114,7 +114,10 @@ def invoke(area=None, resource=None,
         logger.info('Content type header is None.')
         is_content_available = False
     elif 'json' in response.headers.get("content-type") and not out_file:
-        return response.json()
+        response_dict = response.json()
+        response_dict["continuation_token"] = response.continuation_token
```

Comment:
Since we are adding the token to the response in `client.py` do we still need this?